scylla-cdc/src/log_reader.rs

```diff
@@ -17,7 +17,7 @@ pub struct CDCLogReader {
     // Tells the worker to stop
     // Usage of the "watch" channel will make it possible to change the the timestamp later,
     // for example if somebody loses patience and wants to stop now not later
-    end_timestamp: tokio::sync::watch::Sender<chrono::Duration>,
+    end_timestamp: tokio::sync::watch::Sender<Option<chrono::Duration>>,
```

Comment:
Perhaps a better idea than changing the type to `Option<Duration>` is to provide `Duration::max_value()` or something like that instead of `None`.